tests/integration/workspace_access/test_secrets.py

```diff
@@ -0,0 +1,90 @@
+import logging
+from datetime import timedelta
+
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors.mapping import NotFound
+from databricks.sdk.retries import retried
+from databricks.sdk.service.workspace import AclPermission
+
+from databricks.labs.ucx.workspace_access.groups import GroupManager
+from databricks.labs.ucx.workspace_access.manager import PermissionManager
+from databricks.labs.ucx.workspace_access.secrets import SecretScopesSupport
+
+logger = logging.getLogger(__name__)
+
+
+@retried(on=[NotFound], timeout=timedelta(minutes=3))
+def test_secrets(
+    ws: WorkspaceClient,
+    make_ucx_group,
+    sql_backend,
+    inventory_schema,
+    make_secret_scope,
+    make_secret_scope_acl,
+):
+    """
+    This test does the following:
+    * creates 2 ws groups and secret scopes
+    * assigns ACL permissions to the groups on the secrets
+    * run assessment
+    * migrate groups
+    * verify that the migrated groups has the same permissions on the scopes
+    :return:
+    """
+
+    created_items = []
+    for _ in range(2):
```

Comment:
Create just one group group pair, no need for range(2)

Reply:
fixed